readthedocs/settings/testdocsitalia.py

```diff
@@ -18,7 +18,7 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'USER': '',
         'HOST': '',
-        'PORT': 5433,
+        'PORT': 5432,
```

Comment:
non so se lo vuoi pushare questo, lo cambio sempre in locale anche io comunque :\